scripts/helpers/userdetails.py

```diff
@@ -22,28 +22,28 @@
 name = name.replace("'", r"'\''")
 
 base = cfg.get('ldap_base')
-results = mapnames.values()
+results = list(mapnames.values())
 scope = ldap.SCOPE_SUBTREE
 
 query = num and '%s=%d' % (mapnames['unum'], num) or '%s=%s' % (mapnames['uname'], name)
 
 l = ldap.open(cfg.get('ldap_server'))
 r = l.search(base, scope, query, results)
 
-info = dict(zip(props, (name, num, 'Unknown', 'Unknown', '', '', '', '', 0)))
+info = dict(list(zip(props, (name, num, 'Unknown', 'Unknown', '', '', '', '', 0))))
```

Comment:
`dict(list(...))`

😳 wat?

Reply:
Maybe a dict comprehension like this does the same?
```suggestion
info = {k:v for k,v in zip(props, (name, num, 'Unknown', 'Unknown', '', '', '', '', 0))}
```